cypress/support/commands.ts

```diff
@@ -0,0 +1,25 @@
+// ***********************************************
```

Comment:
remember this and e2e.js as well google.cy.js have to be renamed __not to create another file__
Read again the step please https://github.com/holgiosalos/cypress-training-doc#5-configurando-las-pruebas-con-typescript

Reply:
Fixed